uint8 ctr_mode     #0：空闲  1：位置  2：速度
int16 speed_rpm
int16 hight
uint8 lift_state   # bit0: 下限位触发  bit1:上限位触发  bit3:驱动器异常

